scripts/tests/wipe-and-restart-everything.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Deletes everything, including the database and docker containers
+
+set -euo pipefail
+
+docker compose down
+docker volume rm firezone_postgres-data || true
+docker compose run elixir /bin/sh -c 'cd apps/domain && mix ecto.seed'
+docker compose up -d --build --remove-orphans api web client relay gateway dns.httpbin httpbin
```

Comment:
We don't want to `build` in CI -- these are probably different enough ENVs that this may be more of a local developer convenience thing.